src/ralph/cli.py

```diff
@@ -37,6 +37,24 @@
 BACKENDS = (lambda: DATABASE_BACKENDS + STORAGE_BACKENDS)()
 
 
+class ComaSeparatedKeyValueParamType(click.ParamType):
+    """Coma separated key=value parameter type"""
+
+    name = "key=value,key=value"
+
+    def convert(self, value, param, ctx):
+        """Split value by coma and equal sign to return a dict build with key/value pairs"""
+
+        try:
+            return dict(option.split("=") for option in value.split(","))
```

Comment:
I think I'd better try to infer simple types to the value here (at least `bool`). WDYT?

Reply:
Good idea with bools but what would be the meaning of using `int` or other simple types in value?

Reply:
When the `elasticsearch` client will be instantiated, it expects to receive python types for some arguments, not always strings. Some of them are flags (booleans) but some of them may be integers or floats (such as the `timeout` parameter).

Reply:
Ok! Got it ;)

Reply:
And it's done.